account_vat_on_payment/__openerp__.py

```diff
@@ -73,6 +55,16 @@
         'test/account_invoice_7.yml',
         'test/account_invoice_8.yml',
         'test/account_invoice_9.yml',
+        'test/account_invoice_1_real.yml',
+        'test/account_invoice_2_real.yml',
+        'test/account_invoice_3_real.yml',
+        'test/account_invoice_4_real.yml',
+        'test/account_invoice_5_real.yml',
+        'test/account_invoice_6_real.yml',
+        'test/account_invoice_7_real.yml',
+        'test/account_invoice_8_real.yml',
+        'test/account_invoice_1_real_same_account.yml',
```

Comment:
Yes..that's why i want to check the bank test..i just copied the file to have the same invoices, even if i will change the id and the id of the partner, but i had the same issue with the tests that was already in the module, that's why i added the specific Bank Journal in vouchers...otherwise it didn't get the bank account move line.